examples/todo-list/src/__tests__/integration/todo-list-image.integration.ts

```diff
@@ -0,0 +1,62 @@
+import {expect, toJSON} from '@loopback/testlab';
+import {TodoListApplication} from '../../application';
+import {TodoListImageRepository, TodoListRepository} from '../../repositories';
+import {
+  givenRunningApplicationWithCustomConfiguration,
+  givenTodoListImageInstance,
+  givenTodoListInstance,
+  givenTodoListRepositories,
+} from '../helpers';
+
+describe('TodoListApplication', () => {
+  let app: TodoListApplication;
+  let todoListImageRepo: TodoListImageRepository;
+  let todoListRepo: TodoListRepository;
+
+  before(async () => {
+    app = await givenRunningApplicationWithCustomConfiguration();
```

Comment:
Ideally, integration tests should not depend on the entire application, they should create Repository instances via `new`. Only acceptance tests start the full application so that they can make HTTP requests.

See our docs:
- [Test your repositories against a real database](https://loopback.io/doc/en/lb4/Testing-your-application.html#test-your-repositories-against-a-real-database)
- [Create a test datasource](https://loopback.io/doc/en/lb4/Testing-your-application.html#create-a-test-datasource) 
- and let's not forget to [Clean the database before each test](https://loopback.io/doc/en/lb4/Testing-your-application.html#clean-the-database-before-each-test).

I think we may need few iterations to find the best way how to write the integration tests for example-todo-list repositories. Let's leave this refactor out of scope of this pull request, so that we can land it sooner. Having said that, I'd like to keep https://github.com/strongloop/loopback-next/issues/2633 open until the refactor is done. I edited PR description and changed the keyword "Closes" to "See".

Reply:
I updated the setup in https://github.com/strongloop/loopback-next/pull/3171/commits/0ad1a2345394db738435a0c99ee3c9bd555eb250 wdyt?

Reply:
@nabdelgadir I think the current version is good enough and we can close #2633 as done 👍 (Assuming that all items in Acceptance criteria are done now.)